domain-models-runtime/src/main/java/org/folio/rest/impl/TenantAPI.java

```diff
@@ -175,25 +169,48 @@ public String sqlFile(String tenantId, boolean tenantExists, TenantAttributes te
         });
   }
 
-  /**
-   * @param tenantAttributes parameters like module version that may influence generated SQL
-   * @return the SQL commands to create or upgrade the tenant's schema
-   * @throws IllegalArgumentException when templates/db_scripts/schema.json doesn't exist
-   * @throws TemplateException when processing templates/db_scripts/schema.json fails
-   */
-  public Future<String> sqlFile(Context context, String tenantId, TenantAttributes tenantAttributes) {
-    return tenantExists(context, tenantId)
-    .compose(tenantExists -> sqlFile(context, tenantId, tenantAttributes, tenantExists));
+  Future<Void> saveJob(TenantJob tenantJob, String tenantId, String jobId, Context context) {
+    String table = PostgresClient.convertToPsqlStandard(tenantId) + ".rmb_job";
+    String sql = "INSERT INTO " + table + " VALUES ('" + jobId + "', '" + Json.encode(tenantJob) + "'::JSONB)";
+    return postgresClient(context).execute(sql).mapEmpty();
+  }
+
+
+  Future<TenantJob> getJob(String tenantId, String jobId, Context context) {
+    String table = PostgresClient.convertToPsqlStandard(tenantId) + ".rmb_job";
+    String sql = "SELECT jsonb FROM " + table + " WHERE id = '" + jobId + "'";
+    return postgresClient(context).selectSingle(sql)
```

Comment:
Avoid SQL injection, use selectSingle(sql, Tuple.of(jobId)) with prepared/parameterized statement.

Reply:
Thx. Very bad.